test/Setup.t.sol

```diff
@@ -46,6 +46,7 @@ abstract contract Setup is Test {
     address constant owner = address(420);
     address constant refundAddress = address(420420);
```

Comment:
nit: naming style for constants `caller`, `owner` and `refundAddress` are not consistent with others